client.go

```diff
@@ -45,6 +48,26 @@ import (
 )
 
 const defaultStorageURL = "https://api.e3db.com"
+const SECRET_UUID = "38bb737a-4ce0-5ead-8585-e13ea23b09a6"
+const PUBLIC_SIGNING_LENGTH = 43
+const PRIVATE_SIGNING_LENGTH = 86
+const PUBLIC_KEY_LENGTH = 43
+const PRIVATE_KEY_LENGTH = 43
```

Comment:
all of these constants should come from e3db-clients-go(except the SECRET_UUID, thats very SDK specific) , and we should callout the curve and purpose for each of these as well in their naming (e.g. PUBLIC_ENCRYPTION_KEY_LENGTH) https://github.com/tozny/e3db-clients-go/blob/master/crypto.go#L25, cause its not at all apparent to me why the string lengths are different from the byte length (but I'm a crypto noob so my ignorance itself is not that surprising)

Reply:
moved to e3db-clients-go: https://github.com/tozny/e3db-clients-go/pull/178